flask_resty/sorting.py

```diff
@@ -7,11 +7,21 @@
 
 class SortingBase(object):
     def sort_query(self, query, view):
+        return self.sort_query_by_fields(
+            query, view, self.get_request_field_orderings(),
```

Comment:
still need `query`?

Reply:
No, why would I?

Reply:
So let's remove it from the params

Reply:
wait we still need the query to return the sorted query

Reply:
sorry misread